docs/basicusage.rst

```diff
@@ -9,8 +9,7 @@ Minimal Case
 ------------
 
 For the most basic ThreatIngestor setup, you will want to configure at least
-one :ref:`source <source-plugins>`, one :ref:`operator <operator-plugins>`, and
-whether you want to run in daemon or job mode.
+one source_, one operator_, and the daemon_.
```

Comment:
This sounds better. The "and the daemon" part isn't exactly right, what about "and the ``main`` section", or something? The "ingestor" runs in "daemon" mode or "job" mode as defined by that config option. It isn't a daemon, it just optionally daemonizes itself.